packages/geoadmin-elevation-profile/src/profile.api.ts

```diff
@@ -0,0 +1,349 @@
+import type { CoordinatesChunk, CoordinateSystem, SingleCoordinate } from '@geoadmin/coordinates'
+
+import { LV95, removeZValues } from '@geoadmin/coordinates'
+import { log } from '@geoadmin/log'
+import axios from 'axios'
+import proj4 from 'proj4'
+
+import { BASE_URL_DEV, BASE_URL_INT, BASE_URL_PROD, type Staging } from '@/config'
+import getProfileMetadata, { type ElevationProfileMetadata } from '@/utils'
+
+export class ElevationProfileError extends Error {
+    public readonly technicalError: Error
+
+    constructor(message: string, technicalError: Error) {
+        super(message)
+        this.technicalError = technicalError
+    }
+}
+
+type ServiceProfileAltitudes = {
```

Comment:
Will there ever be an advantage if we export these here too?

Reply:
I don't think it's used anywhere else, it's more of a "let's better describe what the backend I encapsulate does" type more than anything